app/views/jst/conferences/concludedConference.handlebars

```diff
@@ -25,9 +25,9 @@
         {{#if recording}}
           <div class="ig-details__item">
             {{#if multipleRecordings}}
-              {{#t "recordings"}}{{recordingCount}} Recordings{{/t}}
+              <div>{{recordingCount}}&nbsp;{{#t "recordings"}}Recordings{{/t}}</div>
             {{else}}
-              {{#t "recording"}}1 Recording{{/t}}
+              <div>1&nbsp;{{#t "recording"}}Recording{{/t}}</div>
```

Comment:
this is incorrect, the count needs to be inside the translation call

Reply:
I'll resore the previous code